bbot/test/test_step_2/module_tests/test_module_discord.py

```diff
@@ -4,15 +4,15 @@
 
 
 class TestDiscord(ModuleTestBase):
-    targets = ["http://127.0.0.1:8888/cookie.aspx", "http://127.0.0.1:8888/cookie2.aspx"]
+    targets = ["http://127.0.0.1:8888/cookie.aspx", "http://127.0.0.1:8888/cookie2.aspx", "foo.bar"]
     modules_overrides = ["discord", "excavate", "badsecrets", "httpx"]
 
     webhook_url = "https://discord.com/api/webhooks/1234/deadbeef-P-uF-asdf"
     config_overrides = {"modules": {"discord": {"webhook_url": webhook_url}}}
 
     def custom_setup(self, module_test):
         respond_args = {
-            "response_data": '<html><body><p>Express Cookie Test<a href="ftp://127.0.0.1/asdf.txt"/></p></body></html>',
+            "response_data": '<html><body><p>Express Cookie Test<a href="ftp://asdf.foo.bar/asdf.txt"/></p></body></html>',
             "headers": {
```

Comment:
Why was this necessary? Are we unable to excavate URLs with IPs?

Reply:
URL excavation with IPs is a must-have feature before we merge.

Reply:
IP urls work not. I honestly can't remember why i made that change in the tests at this point. But i created a new test that explicitly verifies it:

https://github.com/blacklanternsecurity/bbot/pull/1471/commits/3e52d624e44dacc6f9faa4956173db573134367d